### ISSUE TITLE
redis_queue service restart

### ISSUE THREAD

**Issue by satkunas** [inverse-inc/packetfence#6867]
**Describe the bug**
Restarting `redis_queue` produces an error

`POST /api/v1/service/redis_queue/restart ` with payload `{async: true}`

Polling with task_id:

`GET /api/v1/pfqueue/task/ApiTask:36D348D0-85CD-11EC-B8CF-5EB0D8505FB5/status/poll`

Restart completes, the process has a new PID, but the response includes the error:

```json
{"message":"Unable to check if job status exists in redis database","status":500}
```

Furthermore, restarting `redis_queue` synchronously on the same host with another service (eg: `windbindd`) produces a payload where the error is included in the response more than once:

```json
{"message":"Unable to get job status from redis database","status":500}{"message":"Unable to get job status from redis database","status":500}
```

**Expected behavior**
* pfqueue polling should not return an error when the service command succeeds.
* pfqueue response error should be singular, even when run synchronously with other service commands.

**Comment by jrouzierinverse** [comment]
async uses redis_queue to store the result.
This is expected. How should we approach this?

**Comment by julsemaan** [comment]
IMO let's just make it so that redis_queue cannot be individually restarted from the admin (like haproxy-admin) and make it part of the protected services

**Comment by jrouzierinverse** [comment]
This maybe fixed by fa326af6c8d47941176f4d2f86e45828f53b0d4b
